tsconfig.json

```diff
@@ -11,8 +11,6 @@
     { "path": "packages/remix-dev" },
     { "path": "packages/remix-express" },
     { "path": "packages/remix-netlify" },
-    // Commenting out for now because of https://github.com/remix-run/remix/pull/3104#issuecomment-1331779471
```

Comment:
Removed for now because of https://github.com/remix-run/remix/pull/3104#issuecomment-1331779471 and also https://github.com/remix-run/remix/pull/3104#discussion_r1036365421see